### DISCUSSION TITLE
Dependency injection via Microsoft.Azure.Functions.Extensions.DependencyInjection.FunctionsStartup and IFunctionsHostBuilder

### DISCUSSION THREAD

**Discussion by thrixton** [kedacore/keda#867] [General] [answered]
In Azure functions, Dependency injection is supported per [this](https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-dependency-injection) article.
Is this supported in KEDA?

**Comment by tomkerkhove** [comment, marked as answer]
KEDA is fully independent from Azure Functions; we solely scale workloads on Azure regardless of its flavor.

Your question is better asked on the Azure Function side; if it is, then it is supported in KEDA.

**Reply by thrixton** [reply]
Right you are, thanks.
